guide/packages.md

```diff
@@ -28,27 +28,31 @@ The Membrane docker ([Docker Hub](https://hub.docker.com/r/membrane/membrane), [
 | `unifex`                     | Tool automatically generating NIF and CNode interfaces between C/C++ and Elixir | [![Hex.pm](https://img.shields.io/hexpm/v/unifex.svg)](https://hex.pm/packages/unifex) [![Docs](https://img.shields.io/badge/api-docs-yellow.svg?style=flat)](https://hexdocs.pm/unifex) [![GitHub](https://api.iconify.design/octicon:logo-github-16.svg?color=gray&height=20)](https://github.com/membraneframework/unifex)                                                                                 |
 | `bunch`                      | Extension of Elixir standard library                                            | [![Hex.pm](https://img.shields.io/hexpm/v/bunch.svg)](https://hex.pm/packages/bunch) [![Docs](https://img.shields.io/badge/api-docs-yellow.svg?style=flat)](https://hexdocs.pm/bunch) [![GitHub](https://api.iconify.design/octicon:logo-github-16.svg?color=gray&height=20)](https://github.com/membraneframework/bunch)                                                                                     |
 | `sebex`                      | The ultimate assistant in Membrane Framework releasing & development            | [![GitHub](https://api.iconify.design/octicon:logo-github-16.svg?color=gray&height=20)](https://github.com/membraneframework/sebex)                                                                                                                                                                                                                                                                           |
-| `Beamchmark`                 | Tool for measuring EVM performance.                                             | [![Hex.pm](https://img.shields.io/hexpm/v/beamchmark.svg)](https://hex.pm/packages/beamchmark) [![Docs](https://img.shields.io/badge/api-docs-yellow.svg?style=flat)](https://hexdocs.pm/beamchmark) [![GitHub](https://api.iconify.design/octicon:logo-github-16.svg?color=gray&height=20)](https://github.com/membraneframework/beamchmark)                                                                 |
+| `Beamchmark`                 | Tool for measuring EVM performance                                             | [![Hex.pm](https://img.shields.io/hexpm/v/beamchmark.svg)](https://hex.pm/packages/beamchmark) [![Docs](https://img.shields.io/badge/api-docs-yellow.svg?style=flat)](https://hexdocs.pm/beamchmark) [![GitHub](https://api.iconify.design/octicon:logo-github-16.svg?color=gray&height=20)](https://github.com/membraneframework/beamchmark)                                                                 |
 
 ## Plugins
```

Comment:
Perhaps we could write somewhere that we mean when we say "pluggins" (that is - that we mean packages which deliver Membrane's elements and bins , that can be inserted into Membrane pipelines)